roles/ceph-common/tasks/installs/install_on_debian.yml

```diff
@@ -7,49 +7,15 @@
     cache_valid_time=3600
   with_items: debian_package_dependencies
 
-- name: install the ceph repository stable key
-  apt_key: >
-    data="{{ lookup('file', '../../files/cephstable.asc') }}"
-    state=present
-  when: ceph_stable
-
-- name: install the ceph development repository key
-  apt_key: >
-    data="{{ lookup('file', '../../files/cephdev.asc') }}"
-    state=present
-  when: ceph_dev
-
-- name: install intank ceph enterprise repository key
-  apt_key: >
-    data="{{ lookup('file', '../../files/cephstableice.asc') }}"
-    state=present
-  when: ceph_stable_ice
-
-- name: add ceph stable repository
-  apt_repository: >
-    repo="deb http://ceph.com/debian-{{ ceph_stable_release }}/ {{ ceph_stable_distro_source | default(ansible_lsb.codename) }} main"
-    state=present
-  changed_when: false
-  when: ceph_stable
-
-- name: add ceph development repository
-  apt_repository: >
-    repo="deb http://gitbuilder.ceph.com/ceph-deb-{{ ansible_lsb.codename }}-x86_64-basic/ref/{{ ceph_dev_branch }} {{ ansible_lsb.codename }} main"
-    state=present
-  changed_when: false
-  when: ceph_dev
-
-- name: add inktank ceph enterprise repository
-  apt_repository: >
-    repo="deb file://{{ ceph_stable_ice_temp_path }} {{ ansible_lsb.codename }} main"
-    state=present
-  changed_when: false
-  when: ceph_stable_ice
+- name: configure Ceph APT repository
```

Comment:
could just remove the capital letters? so we keep this consistent with the rest of the playbook.